data_structures/stacks/infix_to_prefix_conversion.py

```diff
@@ -83,7 +100,11 @@ def infix_2_postfix(infix):
             if len(stack) == 0:
                 stack.append(x)  # If stack is empty, push x to stack
             else:  # while priority of x is not > priority of element in the stack
-                while len(stack) > 0 and priority[x] <= priority[stack[-1]]:
+                while (
+                    len(stack) > 0
```

Comment:
This is slightly safer if `stack` is set to `None`
```suggestion
                    stack
```